vast.py

```diff
@@ -924,6 +983,12 @@ def change__bid(args: argparse.Namespace):
     r.raise_for_status()
     print("Per gpu bid price changed".format(r.json()))
 
+    if (args.schedule):
+        cli_command = "change bid"
+        api_endpoint = "/api/v0" + "/instances/bid_price/{id}/".format(id=args.id)
```

Comment:
why not reuse the url variable that is equivalent to api_endpoint?

Reply:
url also includes hostname, port maybe, etc.